python/cudf/cudf/core/udf/strings_typing.py

```diff
@@ -27,6 +29,17 @@ def return_type(self):
         return self
 
 
+class ManagedUDFString(types.Type):
+    np_dtype = np.dtype("object")
+
+    def __init__(self):
+        super().__init__(name="managed_udf_string")
+
+    @property
+    def return_type(self):
```

Comment:
Could I suggest we rename this as something like `udf_return_type` or `return_as`? The name is overloaded with the `return_type` of a signature, and it's quite hard to figure out what the use of this `return_type` is going to be. I eventually discovered one use, in `ApplyKernelBase._get_udf_return_type()`, and I think that's all - but I may have missed more?